### ISSUE TITLE
Why is "model. Eval" not used in the test part of the code

### ISSUE THREAD

**Issue by Yudsuki** [yanx27/GACNet#8]
Hi Yan,

Thank you for your fantastic work. But I have some questions.
I found that 'batchnorm' and 'DROPOUT' were used extensively in the code. I implemented the model 40 classification task based on your code. Of course, 'model. train' is used reasonably, but I found that 'model. eval' is not used in part of the test code. What's more strange is that after adding 'model. Eval()', test accuracy will be greatly reduced. Do you encounter this problem during debugging?

**Comment by littlelittlebin** [comment]
i meet the same problem, it is because the maxtrix a?

**Comment by MichaleWong** [comment]
I think he didn't test the network seriously, just run training successfully, and end.

**Comment by MichaleWong** [comment]
actually, after testing , there's difference between eval mode and train mode, that was caused by batch norm mainly
